diff --git a/defaults.go b/defaults.go
index 01199e8..b039077 100644
--- a/defaults.go
+++ b/defaults.go
@@ -35,6 +35,23 @@ var symbolTemplates = map[string]Symbols{
 		RepoConflicted: "\u273C",
 		RepoStashed:    "\u2691",
 	},
+	"flame": {
+		Lock:                 "\uE0A2",
+		Network:              "\uE0A2",
+		Separator:            "\uE0C0",
+		SeparatorThin:        "\uE0C1",
+		SeparatorReverse:     "\uE0C2",
+		SeparatorReverseThin: "\uE0C3",
+
+		RepoDetached:   "\u2693",
+		RepoAhead:      "\u2B06",
+		RepoBehind:     "\u2B07",
+		RepoStaged:     "\u2714",
+		RepoNotStaged:  "\u270E",
+		RepoUntracked:  "+",
+		RepoConflicted: "\u273C",
+		RepoStashed:    "\u2691",
+	},
 	"flat": {
 		RepoDetached:   "\u2693",
 		RepoAhead:      "\u2B06",
